samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -544,6 +584,39 @@ For example, when building from the command line, you can add it as follows:
 
    west build -b *board_target* -- -DFILE_SUFFIX=release
 
+DFU build with the key storage in KMU
+=====================================
+
+The MCUboot-based targets that enable the ``SB_CONFIG_MCUBOOT_SIGNATURE_USING_KMU`` Kconfig option use the Key Management Unit (KMU) hardware peripheral to store the public key that is used by the bootloader to verify the application image.
+The build targets based on the nRF54L SoC Series are currently the only targets that support the KMU-based key storage.
+
+Using KMU requires the provisioning operation of the public key to be performed manually.
+Before performing the provisioning operation, you need to ensure that your build target is fully erased:
+
+.. parsed-literal::
+   :class: highlight
+
+   nrfutil device erase --all
+
+Assuming that your currently working directory points to this sample directory, you can perform the provisioning operation as follows:
+
+.. parsed-literal::
+   :class: highlight
+
+   west ncs-provision upload -s <soc> -k sysbuild/configuration/<build_target>/boot_signature_key_file_<algorithm>.pem --keyname UROT_PUBKEY
+
+The ``<soc>`` placeholder is the SoC name used in your build target (for example, ``nrf54l15``), the ``<build_target>`` placeholder is your build target name (for example, ``nrf54l15dk_nrf54l15_cpuapp``), and ``<algorithm>`` is the algorithm used to generate the application image key pair (for example, ``ed25519``).
```

Comment:
```suggestion
* The ``<soc>`` placeholder is the SoC name used in your board target (for example, ``nrf54l15``).
* The ``<build_target>`` placeholder is your board target name (for example, ``nrf54l15dk_nrf54l15_cpuapp``).
* ``<algorithm>`` is the algorithm used to generate the application image key pair (for example, ``ed25519``).
```
Maybe you should use also <board_target> for the name, both in the command example and in the explanation.

Reply:
I would keep <algorithm> placehold to keep this information general enough so that it is still correct when we add support for other signature algorithms. KMU is also available in the nRF53 so it may as well be used in the MCUboot bootloader pretty soon

Reply:
I would not insist as it's not an error